CHANGELOG.rst

```diff
@@ -2,15 +2,18 @@
 Change history
 ==============
 
-2.0.0 (2024-11-??)
+2.0.0 (2024-11-22)
 ------------------
 
 * upgrade to zgw-consumers 0.35.1
-* remove zds-client dependency
+* remove zds-client dependency and replace with ``ape_pie.APIClient``
```

Comment:
I think adding [this](https://github.com/maykinmedia/commonground-api-common/pull/38#discussion_r1853559223) comment here would make live easier for developers that upgrade.